File: packages/insomnia-httpsnippet/test/fixtures/output/java/okhttp/full.java
OkHttpClient client = new OkHttpClient();

MediaType mediaType = MediaType.parse("application/x-www-form-urlencoded");
RequestBody body = RequestBody.create(mediaType, "foo=bar");
Request request = new Request.Builder()
  .url("http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value")
  .post(body)
  .addHeader("cookie", "foo=bar; bar=baz")
  .addHeader("accept", "application/json")
  .addHeader("content-type", "application/x-www-form-urlencoded")
  .build();

Response response = client.newCall(request).execute();

File: packages/insomnia-httpsnippet/test/fixtures/output/node/request/full.js
var request = require('request');

var jar = request.jar();
jar.setCookie(request.cookie('foo=bar'), 'http://mockbin.com/har');
jar.setCookie(request.cookie('bar=baz'), 'http://mockbin.com/har');

var options = {
  method: 'POST',
  url: 'http://mockbin.com/har',
  qs: { foo: ['bar', 'baz'], baz: 'abc', key: 'value' },
  headers: {
    'content-type': 'application/x-www-form-urlencoded',
    accept: 'application/json'
  },
  form: { foo: 'bar' },
  jar: 'JAR'
};

request(options, function(error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});
